dojo/utils.py

```diff
@@ -344,10 +346,13 @@ def set_duplicate(new_finding, existing_finding):
 
 
 def is_duplicate_reopen(new_finding, existing_finding):
-    if (existing_finding.is_Mitigated or existing_finding.mitigated) and not existing_finding.out_of_scope and not existing_finding.false_p and new_finding.active and not new_finding.is_Mitigated:
-        return True
-    else:
-        return False
+    return (
+        (existing_finding.is_Mitigated or existing_finding.mitigated) and
+        not existing_finding.out_of_scope and
+        not existing_finding.false_p and
+        new_finding.active and
+        not new_finding.is_Mitigated
+    )
 
```

Comment:
@danielnaab try to switch to flake8 linter as the CI pipeline check syntax iwth flake8
@valentijnscholten in VS code you can select the linter. I'm using VS code also with flake8 configured for the project and it works well (same warning in VS code as in the CI pipelines)

Reply:
I also use flake8 in vscode. And the new pylance, it's an improvement!

Reply:
It might in part be a matter of the `.flake8` config in the codebase, but flake8 is missing a lot of structural issues - undefined variables, etc. pylint found several such bugs missed by flake8.